recipes/grpc-cpp/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "grpc-cpp" %}
+{% set version = "1.18.0" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/grpc/grpc/archive/v{{ version }}.tar.gz
+    sha256: 069a52a166382dd7b99bf8e7e805f6af40d797cfcee5f80e530ca3fc75fd06e2
+    patches:
+      # Building the Objective C plugin fails on Windows, skip it.
+      - 0001-remove-objective-c-target.patch  # [win]
+
+build:
+  number: 1000
+  # Requires C++ 11, VS 2008 is not supported
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    # `protoc` is also used for building
+    - libprotobuf 3.6.1
+    - ninja
+  host:
+    - c-ares
+    - gflags
+    - libprotobuf 3.6.1
+    - openssl
+    - zlib
+  run:
+    # Need to pin libprotobuf to the same version used when building,
+    # otherwise we may get a different SO ABI version.
+    - libprotobuf 3.6.1
```

Comment:
This should be replaced by `{{ pin_compatible("libprotobuf", max_pin="x.x.x") }}`

Reply:
If I pass "x.x.x", will it ensure than the right bugfix version is depended upon at runtime (FTR, 3.6.0 is ABI-incompatible with 3.6.1...)?

Reply:
Yes